bundle/manifests/cluster-logging.v5.0.0.clusterserviceversion.yaml

```diff
@@ -122,7 +122,7 @@ spec:
     must be explicitly created by a cluster administrator (e.g. `oc create ns openshift-logging`). To enable metrics
     service discovery add namespace label `openshift.io/cluster-monitoring: "true"`.
 
-    For additional installation documentation see [Deploying cluster logging](https://docs.openshift.com/container-platform/4.7/logging/cluster-logging-deploying.html)
+    For additional installation documentation see [Deploying cluster logging](https://docs.openshift.com/container-platform/5.0/logging/cluster-logging-deploying.html)
```

Comment:
@rolfedh what is the correct URL for this documentation given OCP 5.0 does not exist yet?